architecture-decision-record/0027-use-member-cicd-access-for-configuration-management.md

```diff
@@ -0,0 +1,41 @@
+# 27. Use Member CICD Access for Configuration Management
+
+Date: 2023-08-17
+
+## Status
+
+✅ Accepted
+
+## Context
+
+Users of the Modernisation Platform often host applications which require additional configuration on top of standard infrastructure provisioning.
+
+This configuration whilst sometimes manual often can be achieved via code, eg scripts or running a configuration management tool such as Ansible on the server.
+
+## Options
+
+#### Run configuration management tools manually on servers
+
+This works but is not best practice and runs the risk of things not being applied consistently.
+
+#### Run configuration management from the environments repository pipeline
+
+Automated solution and keeps everything in one place, but begins to mix infrastructure with configuration which we do not want to do. The environments repository is already very large and this will add to this.
+
+#### Run configuration management with another platform managed repository
+
+Automated solution and keeps infrastructure and configuration separate, however this will be hard to create something which works with different configuration management tools.  Different applications have different requirements, with infrastructure it is simpler as it uses one tool - Terraform.  One application may required Ansible, another may need to make calls to an AWS API.  This solution will add considerable overhead to the Modernisation Platform team.
```

Comment:
again first sentence does not read right to me whats automated would it not read better with something like an automated solution that keeps

Reply:
also maybe replace which works with that works

Reply:
also maybe move the comma after different requirements, to it is simpler, i think it flows better

Reply:
may required change to require